### PR TITLE
More simple cleanups

### PR DESCRIPTION
**Pull request by maage**
Three style cleanups and another three changes.

### CODE REVIEW

**Comment by jedisct1** [comment]
👍

File: dnscrypt-proxy/proxy.go [hunk 1]
```diff
@@ -417,7 +415,6 @@ func (proxy *Proxy) processIncomingQuery(serverInfo *ServerInfo, clientProto str
 					sharedKey, encryptedQuery, clientNonce, err := proxy.Encrypt(serverInfo, query, serverProto)
 					if err != nil {
 						pluginsState.returnCode = PluginsReturnCodeParseError
-						pluginsState.ApplyLoggingPlugins(&proxy.pluginsGlobals)
```
Comment (jedisct1) [inline dnscrypt-proxy/proxy.go:420]: With the `defer`, that case may not be handled any more.

Comment (maage) [reply dnscrypt-proxy/proxy.go:420]: Shouldn't it fail there?

Comment (jedisct1) [reply dnscrypt-proxy/proxy.go:420]: Actually, yes, it should fail :)

**Pull request merged**